src/libs/OptionsListUtils.ts

```diff
@@ -1625,131 +1669,142 @@ function getValidReports(reports: OptionList['reports'], config: GetValidReports
         includeSelfDM = false,
         includeInvoiceRooms = false,
         action,
-        selectedOptions = [],
         includeP2P = true,
         includeDomainEmail = false,
-        shouldBoldTitleByDefault = true,
         loginsToExclude = {},
-        shouldSeparateSelfDMChat,
-        shouldSeparateWorkspaceChat,
         excludeNonAdminWorkspaces,
-        isPerDiemRequest = false,
-        showRBR = true,
     } = config;
     const topmostReportId = Navigation.getTopmostReportId();
 
-    const validReportOptions: OptionData[] = [];
-    const workspaceChats: OptionData[] = [];
-    let selfDMChat: OptionData | undefined;
-    const preferRecentExpenseReports = action === CONST.IOU.ACTION.CREATE;
+    // eslint-disable-next-line rulesdir/prefer-at
+    const option = reportOption;
```

Comment:
Also, is this line necessary at all? Can we remove the redundant `option` variable here (and maybe rename `reportOption` to `option` for brevity)